e2e_test/streaming/datagen.slt

```diff
@@ -45,6 +45,9 @@ create materialized source s1 (v1 int)  with (
 # Wait enough time to ensure Datagen connector generate data
 sleep 2s
 
+statement ok
+flush;
+
```

Comment:
Aadding `flush` is  to add a checkpoint here, so that the data generated can be read.